src/App.js

```diff
@@ -36,7 +37,15 @@ const App = () => {
       <div className="container-fluid">
         <header className="App-header">
           <nav className="App-logo">
-            <img src="./logo512.png" width="192" height="192" alt="logo"></img>
+            <a href="/">
```

Comment:
Logo must change state (size) from mobile to desktop? I think this is a thing that works, right?

Reply:
Here I just added an <a> around the logo so it will redirect to "/" when clicked.

Reply:
yeah. It  should redirect to home (sometimes this is done with a HomeIcon somewhere in the topnav.  Somewhere I was reading about logo size state change when on mobile vs. desktop.. thats why react provided 192 and 512 sizes.  So the logo size should be dynamic.